plans/tier0.fmf

```diff
@@ -178,11 +178,11 @@ description+: |
 
 
     /convert_offline_systems:
-        # At the moment we have only CentOS7 repos available on Satellite server
         adjust+:
-            - enabled: false
-              when: >
-                distro != centos-7
+          # Disabled due to known issue: https://issues.redhat.com/browse/RHELC-1040
+          # Enable once this gets fixed
+          - enabled: false
+            when: distro == oraclelinux-7
```

Comment:
```suggestion
            when: distro == oraclelinux-7, centos-7
```
CentOS-7 suffers the same fate, not resolving to `7server`, we should disable for both until resolved.